### PR TITLE
Add pybuilder recipe

### PR DESCRIPTION
**Pull request by willprice**

### CODE REVIEW

**Comment by willprice** [comment]
@conda-forge/staged-recipes could you review this please? I think I can ignore the azure windows build? If not let me know.

File: recipes/pybuilder/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ea740e0c4794935a401b4f19d67b34c660d6f0147bb135c6918a5c8c563dade9
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: True  # [py==37]
```
Comment (scopatz) [inline recipes/pybuilder/meta.yaml]: I think it is better to use the `py37` selector here.  The `py` variable has historically changed type (from int to str) and so I think of it as less stable.

Comment (willprice) [reply recipes/pybuilder/meta.yaml]: I thought `py37` was discouraged over `py==37`: https://github.com/conda/conda-build/pull/3363/files

Comment (scopatz) [reply recipes/pybuilder/meta.yaml]: Fair, mike knows better

**Pull request merged**